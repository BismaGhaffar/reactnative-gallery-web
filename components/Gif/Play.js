import styled from 'styled-components'

const Play = styled.div`
  @media (max-width: 769px) {
    position: absolute;
    z-index: 100;
    margin: 0px auto;
    width: 100%;
    height: 100%;
    max-width: 275px;
    opacity: 0.5;
    content: url(/static/images/play-circle.svg);
    display: ${props => (props.show ? 'block' : 'none')};
  }
`

export default Play
