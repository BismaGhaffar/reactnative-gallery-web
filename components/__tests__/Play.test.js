import React from 'react'
import renderer from 'react-test-renderer'
import Play from '../Gif/Play'

it('Play can be created', () => {
  const comp = renderer.create(<Play />)
  expect(comp).toBeDefined()
})

it('<Play /> toMatchSnapshot', () => {
  const tree = renderer.create(<Play />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('<Play /> toMatchSnapshot 2', () => {
  const tree = renderer.create(<Play show />).toJSON()
  expect(tree).toMatchSnapshot()
})
