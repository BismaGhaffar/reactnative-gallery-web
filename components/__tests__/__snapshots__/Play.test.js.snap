// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Play /> toMatchSnapshot 1`] = `
<div
  className="Play-s10ikt1c-0 dcOaWh"
/>
`;

exports[`<Play /> toMatchSnapshot 2 1`] = `
<div
  className="Play-s10ikt1c-0 gLlmwW"
/>
`;
